apps/aeutil/lib/patricia_merkle_tree.ex

```diff
@@ -86,12 +87,54 @@ defmodule Aeutil.PatriciaMerkleTree do
   """
   @spec verify_proof(Trie.t(), Trie.key(), Trie.value(), Trie.t()) :: boolean
   def verify_proof(trie, key, value, proof) do
-    Proof.verify_proof(key, value, trie.root_hash, proof.db)
+    Proof.verify_proof(key, value, trie.root_hash, proof)
   end
 
   @doc """
   Deleting a value for given key and reorganizing the trie
   """
   @spec delete(Trie.t(), Trie.key()) :: Trie.t()
   def delete(trie, key), do: Trie.delete(trie, key)
+
+  @doc """
+  Providing pretty print of a given trie in the shell.
+  Depending on the atom it can print structure or key value pairs
+
+  ## Examples
+
+  If we want to print as pair
+      iex> Aeutil.PatriciaMerkleTree.new(:test_trie) |> Aeutil.PatriciaMerkleTree.enter("111", "val1") |> Aeutil.PatriciaMerkleTree.enter("112", "val2") |> Aeutil.PatriciaMerkleTree.print_trie(:as_pair)
```

Comment:
I think one example to print one chain state tree would be sufficient

Reply:
> I think one example to print one chain state tree would be sufficient

After #366 is merged, I can provide such an example.